ext/datastore/bigquery/adapter_test.go

```diff
@@ -67,6 +67,34 @@ func TestAdapter(t *testing.T) {
 		},
 	}
 
+	t.Run("should convert from and to BQ ExternalDataConfig successfully", func(t *testing.T) {
+		externalDataSource := BQExternalSource{
+			SourceType: string(ExternalTableTypeGoogleSheets),
+			SourceURIs: []string{"http://googlesheets.com/1234"},
+			Config:     map[string]interface{}{"skip_leading_rows": 1.0, "range": "A!:A1:B1"},
+		}
+		expectedBQExternalDataConfig := &bigquery.ExternalDataConfig{
+			SourceFormat: bigquery.GoogleSheets,
+			SourceURIs:   []string{"http://googlesheets.com/1234"},
+			Options: &bigquery.GoogleSheetsOptions{
+				SkipLeadingRows: 1,
+				Range:           "A!:A1:B1",
+			},
+		}
+
+		bQExternalDataConfigResult, err := bqExternalDataConfigTo(externalDataSource)
+
```

Comment:
can you please format to remove these empty new lines in between?

Reply:
Will refactor this